yarn-project/noir-libs/noir-aztec/src/note/lifecycle.nr

```diff
@@ -29,6 +31,23 @@ fn create_note<Note, N>(
     context.push_new_note_hash(inner_note_hash);
 }
 
+fn create_note_hash_from_public<Note, N>(
+    inputs: PublicContextInputs,
+    storage_slot: Field,
+    note: &mut Note,
+    note_interface: NoteInterface<Note, N>,
+) {
+    let mut inner_note_hash = 0;
```

Comment:
This does not need to be initialised up here, it can just be assigned on line 46

Reply:
Good catch. I'll fix `create_note` too.